sphinx_gallery/directives.py

```diff
@@ -393,8 +393,7 @@ def _copy_images(self, node):
     # copy image from source to imagedir.  This is
     # *probably* supposed to be done by a builder but...
     # ie myproj/build/html/_images
-    imagedir = os.path.join(self.builder.imagedir, "")
-    imagedir = PurePosixPath(self.builder.outdir, imagedir)
+    imagedir = PurePosixPath(self.builder.outdir, self.builder.imagedir)
```

Comment:
`os.path.join(... , "")` does nothing, except maybe adding a "/" to the end of the string, but Path does not care wether the string ends with a "/", so we caln leave this step out.